.env.example

```diff
@@ -13,7 +13,7 @@ Demo="true" #skips userid check to allow access
 
 DefaultProvider="" # mx | sophtron
 
-Port="8080" #the port where the server runs
+PORT="8080" #the port where the server runs
```

Comment:
Do to conventions with GutHub, any ENV var used in GH actions needs to follow standards of being UPPERCASE. That is why some of the env vars have been updated. I propose we change them all in the near future.